test/unit/cuesTest.js

```diff
@@ -0,0 +1,34 @@
+const test = require('tape');
+const DanceParty = require('../../src/p5.dance');
+
+const createDanceAPI = () => {
```

Comment:
Nit: we're starting to duplicate this a number of places, I wonder if its worth extracting to a test/helpers util?

(Non-blocking, we could address this in a follow up PR.)